pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -326,7 +326,7 @@ type TestTemplatesConfigBodyParams struct {
 // swagger:model
 type TestTemplatesResults struct {
 	Results []TestTemplatesResult      `json:"results"`
```

Comment:
I think this should have `omitempty` too:

```suggestion
	Results []TestTemplatesResult      `json:"results,omitempty"`
```
For example:

```
{
  "results": null,
  "errors": [
    {
      "kind": "invalid_template",
      "message": "template: :2: unexpected EOF"
    }
  ]
}
```